include/psp2/promoterutil.h

```diff
@@ -53,13 +68,13 @@ int scePromoterUtilityDeletePkg(const char *titleid);
 int scePromoterUtilityUpdateLiveArea(ScePromoterUtilityLAUpdate *args);
 
 /**
- * Install a import from a directory, and add an icon on the LiveArea.
+ * Install Content Manager import contents and create bubbles without checking license files.
  *
- * @param[in] *path - the path of the directory where the extracted content of the import is
+ * @param[in] *params - see ScePromoterUtilImportParams
```

Comment:
ScePromoterUtilImportParams should have double colon to indicate it's in the global scope ::

the one right above it has this(ScePromoterUtilityLAUpdate)